website/content/api-docs/system/mounts.mdx

```diff
@@ -212,6 +212,23 @@ $ curl \
     http://127.0.0.1:8200/v1/sys/mounts/my-mount
 ```
 
+### Force Disable
+
+Because disabling a secrets engine revokes secrets associated with this mount,
+possible errors can prevent the secrets engine from being disabled if the
+revocation fails.
+
+The best way to resolve this is to figure out the underlying issue and then
+disable the secrets engine once the underlying issue is resolved. Often, this can be as
+simple as increasing the timeout (in the event of timeout errors).
+
+For recovery situations where the secret has been manually removed from the
+secrets backing service, one can force a secrets engine disable in Vault by
+performing a [force revoke](https://www.vaultproject.io/api/system/leases#revoke-force)
+on the mount prefix, followed by a secrets disable when that completes. 
```

Comment:
unsure whether secrets disable needs to be wrapped in code

Reply:
I don't want it here since `secrets disable` is a command and this is for the API